pkgs/applications/kde/kigo.nix

```diff
@@ -0,0 +1,22 @@
+{ mkDerivation, lib, extra-cmake-modules, kdoctools, ki18n, kio, libkdegames, knewstuff }:
+
+mkDerivation {
+  name = "kigo";
+  meta = with lib; {
+    homepage = "https://kde.org/applications/en/games/org/kde.kigo";
+    description = "Kigo est une implémentation libre du jeu de Go";
```

Comment:
@freezeboy - any reason this is in French?

Reply:
wrong copy paste I guess, I should start a new PR from staging ?